### DISCUSSION TITLE
Bootstrap Migration from v3 to v5

### DISCUSSION THREAD

**Discussion by kushgup123** [twbs/bootstrap#39223] [Q&A]
My team will start the POC of migration from Bootstrap v3 to v5. Firstly is that possible ? What are the different kind of roadblocks that the team might face.

**Comment by julien-deramond** [comment]
A migration from Bootstrap v3 to v5 might be tricky because of the 2 major versions difference, but it is possible.
You can refer to the following migration guides to spot the main differences:
- [Migrating to v4](https://getbootstrap.com/docs/4.6/migration/)
- [Migrating to v5](https://getbootstrap.com/docs/5.3/migration/)

For more detail on specific points, you might want to refer to the release notes of the intermediate versions too: https://github.com/twbs/bootstrap/releases.

However, I wouldn't use the v4 as a transition to avoid having to do a double migration. I would try to target the latest Bootstrap version available.
Good luck with this work @kushgup123
